tests/core/mem_growth.cpp

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
```

Comment:
please rename this file to match the python test name, so maybe `emmalloc_memgrowth.cpp`?